### PR TITLE
KZOO-335: add call limits schema

### PR DESCRIPTION
**Pull request by harenson**
Depends on: https://github.com/2600hz/kazoo-core/pull/512

Allows to set a maximum number of concurrent calls (limit) per endpoint (device/user) or account-wide. When building the endpoint, if limit is configured at endpoint level, it takes precedence over account's configuration in case limit was also configured account-wide.

### CODE REVIEW

**Review by danielfinke — APPROVED** [review]

File: priv/couchdb/schemas/call_limits.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "_id": "call_limits",
+    "description": "Schema for call limits",
+    "properties": {
+        "max_concurrent": {
+            "description": "Maximum number of concurrent calls allowed per endpoint",
+            "type": "integer"
+        }
```
Comment (lazedo) [inline priv/couchdb/schemas/call_limits.json:9]: lets add a `direction` property with enum inbound/outbound/both and default to `both`

**Review by lazedo — CHANGES_REQUESTED** [review]

File: priv/couchdb/schemas/call_limits.json (hunk 1 above)
Comment (harenson) [reply priv/couchdb/schemas/call_limits.json:9]: @lazedo should we stick with the existing `[inbound, outbound, any]` values for direction?

Comment (lazedo) [reply priv/couchdb/schemas/call_limits.json:9]: @harenson nm, lets leave it as-is

Comment (harenson) [reply priv/couchdb/schemas/call_limits.json:9]: @lazedo Ok, I will take care of the comments on the other related PRs, then. Thank you.

**Comment by harenson** [comment]
_docs_ check is failing because it depends on https://github.com/2600hz/kazoo-core/pull/512 changes.

**Review by lazedo — APPROVED** [review]

**Pull request merged**